src/constants/error.js

```diff
@@ -0,0 +1,24 @@
+const ERROR_PATTERNS = {
+  NOT_AUTHORIZED: {
+    status: 401,
+    message: "Unauthorised",
```

Comment:
아래 createError에서 사용한 표기와 통일

```suggestion
    message: "Unauthorized",
```

Reply:
오타 수정하겠습니다!